pkg/common/info.go

```
@@ -85,19 +88,27 @@ type UvmInformation struct {
 func GetUvmSecurityCtxDir() (string, error) {
 	securityContextDir := os.Getenv("UVM_SECURITY_CONTEXT_DIR")
 	if securityContextDir == "" {
-		return "", errors.New("UVM_SECURITY_CONTEXT_DIR not set")
+		logrus.Debugf("UVM_SECURITY_CONTEXT_DIR not set.  Using system default %q", uvmSecurityCtxDirDefault)
+		securityContextDir = uvmSecurityCtxDirDefault
+	} else {
+		logrus.Infof("UVM_SECURITY_CONTEXT_DIR is set to %s", securityContextDir)
 	}
-  logrus.Infof("UVM_SECURITY_CONTEXT_DIR is set to %s", securityContextDir)
 	return securityContextDir, nil
 }
 
 func GetUvmInformation() (UvmInformation, error) {
-	_, err := GetUvmSecurityCtxDir()
-	if err != nil {
-		return GetUvmInformationFromEnv()
-	} else {
-		return GetUvmInformationFromFiles()
+	contextDir, err := GetUvmSecurityCtxDir()
+	var info UvmInformation
+	if len(contextDir) > 0 {

```

should this check `err != nil` rather than/in addition to the length of contextDir? it seems like that instance of err isn't referenced anywhere

---

I was actively ignoring the error.  Could use `_` I suppose.

---

I think we can just leave it as is, otherwise we have to declare a `var err error` because we cannot use := on line 103) due to info